ntbs-service/DataMigration/TreatmentEventMapper.cs

```diff
@@ -78,5 +80,31 @@ private async Task TryAddTbServiceAndCaseManagerToTreatmentEvent(TreatmentEvent
                 ev.CaseManagerId = (await _referenceDataRepository.GetUserByUsernameAsync(caseManagerUsername)).Id;
             }
         }
+
+        private string RemoveUnnecessaryNoteInfo(string note)
+        {
+            if (note.IsNullOrEmpty())
+            {
+                return null;
+            }
+            var trimmedNote = note.Trim();
+            var mainPattern =
+                @"(Dear|Hi)[0-9a-zA-Z -]+,{0,1}[\n\r ]*(?<caseManagerText>[0-9a-zA-Z \/\-—,.'`@#&+;:$_()<>\\\[\]=\*\?\n\r]*)"
```

Comment:
Will this work if there are case managers with accents in their name?

Reply:
No I don't think it will play nicely with accents, but it doesn't look like there can be many with accents as this matches nearly all comments.